### PR TITLE
chore(amplify-e2e-tests): update readme and sample env

### PR DESCRIPTION
**Pull request by 0618**
#### Description of changes

- update README and sample env for e2e test and migration test

#### Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-e2e-tests/Readme.md [hunk 1]
```diff
@@ -4,27 +4,27 @@ This packages contains end to end tests that are run in CircleCI to ensure that
 
 ## Setup
 
-To run the tests locally, you need to have your AWS credentials stored in a `.env` file of this package. These values are used to configure the test projects.
+To run the tests locally, you need to have your personal AWS credentials stored in a `.env` file of this package. These values are used to configure the test projects.
 
-Please see sample.env for the keys that are expected in your `.env` file.
+Please see sample.env for the keys that are expected in your `.env` file. Or, you can copy "Copy bash/zsh" directly from Isengard of you personal account, and paste it in the terminal.
```
Comment (Amplifiyer) [inline packages/amplify-e2e-tests/Readme.md:9]: I know it's probably not meant for external contributors to run e2e tests, we should still not expose internal concepts such as `Isengard` in our readmes

File: packages/amplify-e2e-tests/sample.env [hunk 2]
```diff
@@ -1,3 +1,5 @@
+AMPLIFY_PATH=<PATH_TO_ROOT/amplify-cli/bin/amplify>
```
Comment (Amplifiyer) [inline packages/amplify-e2e-tests/sample.env:1]: Isn't `<PATH_TO_ROOT/packages/amplify-cli/bin/amplify>` as per your updated comment?

Comment (0618) [reply packages/amplify-e2e-tests/sample.env]: Good catch. Updated

File: packages/amplify-e2e-tests/Readme.md (hunk 1 above)
Comment (0618) [reply packages/amplify-e2e-tests/Readme.md]: Make sense. Updated.

**Review by joebuono — APPROVED** [review]

**Pull request merged**